### ISSUE TITLE
APERTURE silently assumes CENTRE=False in previous TWISS

### ISSUE THREAD

**Issue by roman-martin** [MethodicalAcceleratorDesign/MAD-X#603]
The following example demonstrates an undocumented pitfall when using TWISS with the option CENTRE=True:

```
!----EXAMPLE-----------------------------------------------------------------------

!----generic test sequences----------------------------------------
KQ1 = 0.01;
KQ2 = -0.01;

testseq: SEQUENCE, l = 50.0;
        MQ1: QUADRUPOLE, K1=KQ1, l= 2, at=15, APERTYPE=CIRCLE, APERTURE={0.02};
        MQ2: QUADRUPOLE, K1=KQ2, l= 2, at=45, APERTYPE=CIRCLE;
ENDSEQUENCE;

BEAM, PARTICLE=PROTON, ENERGY=7000.0, EXN=2.2e-6, EYN=2.2e-6;


!---TWISS with centre=True and APERTURE
USE, SEQUENCE=testseq;
TWISS, BETX=100, BETY=100, file=twiss.out, CENTRE=True;
APERTURE, file=aperture.out;

!---/EXAMPLE-----------------------------------------------------------------------
```

You will find that the sliced quadrupole MQ1 is located at s=14..16m but in the aperture table it appears at s = 13..15m. APERTURE also wrongly calculates e.g. the beta functions for MQ1.

**Comment by tpersson** [comment]
I looked into this a bit more and in case you have CENTRE=TRUE and only COLLIMATOR  have an aperture limitation it all works fine. I also tried with an RBEND but it gave the same behaviour as with the the Quadrupole in the example.

**Comment by tpersson** [comment]
This is fixed in #667.

**Comment by madcern** [comment]
The bug is fixed but in case we want to be able to run with all twiss settings there is still work to be done. I changed it to a possible enhancement.
